packages/@lwc/ssr-runtime/src/lightning-element.ts

```diff
@@ -26,6 +26,7 @@ import { ClassList } from './class-list';
 import { Attributes, Properties } from './types';
 import { mutationTracker } from './mutation-tracker';
 import { reflectAttrToProp, descriptors as reflectionDescriptors } from './reflection';
+import type { Stylesheets } from '@lwc/engine-core';
```

Comment:
Let's put this in `@lwc/shared` so that we don't have to import directly from `engine-core`

Reply:
Done! 1c976809d